server/api/v1.py

```diff
@@ -564,6 +564,29 @@ def rate_review_for_user():
 
     return vote_added_response
 
+@api.route('/user/remove_scholarship_from_profile', methods=['PUT'])
```

Comment:
I agree with you in principle that API auth should not be cookie based, but consistency dominates correctness here IMO. And in this case, this is not meant to act as a third party API, so it's fine. It only starts being a pain when you start trying to support both token-based and cookie-based authentication.